### ISSUE TITLE
Forced Minimum Window Size

### ISSUE THREAD

**Issue by obogorr** [fmauNeko/MarketBoardPlugin#42]
For a while now the MBP is forcing a minimum sized window when it did not do so before.  The screenshot is showing the smallest I can resize to without minimizing it completely.  I can get by *okay* minimizing it with the arrow, but I had to stop using it in tabbed view as the arrow doesn't work there (which I can make a separate issue for if you'd like).  

![MBP](https://user-images.githubusercontent.com/26394800/181177441-81fb2b04-4205-4c5d-ba4e-c71a069536c3.png)

**Comment by Pythyu** [comment]
Uhmm that's weird maybe it's a compatibility issue with other plugins ? if you try to disable all your plugins and keep only the market board plugin are you still having the same issue like arrow that doesn't work and all ?

**Comment by obogorr** [comment]
Disabled everything except Market Board and issue persists.  For clarification, I can make the window large by dragging, and I can hide it completely with the arrow (when not in tabbed view) but I can not make it any smaller than that size.

**Comment by Pythyu** [comment]
Ok, the line that handles the windows minimum size is
```C#
ImGui.SetNextWindowSizeConstraints(new Vector2(700, 450) * scale, new Vector2(10000, 10000) * scale);
```
Meaning the current minimum market board windows size is 700x450 (* FontGlobalScale that may not scale correctly for ya)


Are you by any chance playing FF on a low resolution ? might be why
And if you try to increase your game resolution does it fix it temporarily ? 

To fix this I could just lower the minimum even more or maybe find a way to scale it base on the current game resolution.

**Comment by obogorr** [comment]
While I'm not playing on a low resolution, I do have the UI scaling turned up so I can read things more easily.  Sounds like that scale modifier is likely the problem

**Comment by Pythyu** [comment]
Just for testing purposes : 
- what is your resolution currently ? 
- What is your current UI scale modifier ?
- if you put your UI scale modifier down does it help ?

**Comment by obogorr** [comment]
![image](https://user-images.githubusercontent.com/26394800/181354969-0aa31b2a-9b40-4c2e-bc5c-831305927cf6.png)
these are my settings, and the issue persists even after I tried scaling it down

**Comment by Pythyu** [comment]
What did you scale down the "High resolution UI Settings" or "Default UI Size" if possible try both.
If one of these work it means it come from scale otherwise it doesn't

**Comment by obogorr** [comment]
I did each individually, and then both to no effect.

**Comment by Pythyu** [comment]
uhmm ok I could add a display tab in the new setting menu in #40 allowing user to choose minimum size to see if it helps.
Since i can't reproduce what's happening to ya, if you're down i can give you a modified version of the plugin where the minimum size is halfed just to check if it solves the problem. 
If you're down hit me up on discord : Marethyu#4284 so i can just drag and drop the modified plugin

**Comment by Pythyu** [comment]
Fix implemented in #41

**Comment by fmauNeko** [comment]
Closed by #41
